common/app/common/Preload.scala

```diff
@@ -0,0 +1,14 @@
+package common
+
+import model.ApplicationIdentity
+
+object Preload {
+
+  val articleDefaultPreloads = Seq("content.css", "javascripts/graun.standard.js", "javascripts/graun.commercial.js")
```

Comment:
Can these strings be case classes e.g. `CssPreloadAsset("content")` `JsPreloadAsset("javascripts/graun.standard")`?  That way we don't have to partition the sequence in `withPreload`, we can just do a map + pattern match.  Might be overkill but it would give a way to encapsulate the logic for stripping prefixes or not as well.